.gitattributes

```diff
@@ -0,0 +1,3 @@
+*     text=auto
```

Comment:
```suggestion
* text=auto
```

Reply:
I agree. That's what the [gitattributes](https://git-scm.com/docs/gitattributes) documentation suggests, but it looks ugly. Fixed now.